content/articles/2019-09-tidymodels.Rmarkdown

```diff
@@ -0,0 +1,140 @@
+---
+title: tidymodels updates
+slug: tidymodels-2019-09
+author: Max Kuhn, Edgar Ruiz, and Davis Vaughan 
+description: The latest updates to the tidymodels packages
+date: '2019-09-03'
+categories: [package]
+tags:
+  - tidymodels
+photo:
+  url: https://unsplash.com/photos/rqIfy1UyIzE
+  author: Branden Harvey
+---
+
+We've sent a few packages to CRAN recently. Here's a recap of the changes (and some notes at the bottom): 
+
+## recipes 0.1.6
+ 
+### Breaking Changes
+
+ * Since 2018, a warning has been issued when the wrong argument was used in `bake(recipe, newdata)`. The deprecation period is over and `new_data` is officially required.  
+ * Previously, if [`step_other()`](https://tidymodels.github.io/recipes/reference/step_other.html) did _not_ collapse any levels, it would still add an "other" level to the factor. This would lump new factor levels into "other" when data were baked (as  [`step_novel()`](https://tidymodels.github.io/recipes/reference/step_novel.html) does). This no longer occurs since it was inconsistent with `?step_other`, which said that: "If no pooling is done the data are unmodified".
+ 
+### New Operations:
+
+*  [`step_normalize()`](https://tidymodels.github.io/recipes/reference/step_normalize.html) centers and scales the data (if you are, like Max, too lazy to use two separate steps). 
+*  [`step_unknown()`](https://tidymodels.github.io/recipes/reference/step_unknown.html) will convert missing data in categorical columns to "unknown" and update factor levels. 
+ 
+### Other Changes:
+
+* If `threshold` argument of [`step_other()`](https://tidymodels.github.io/recipes/reference/step_other.html) is greater than one then it specifies the minimum sample size before the levels of the factor are collapsed into the "other" category. [#289](https://github.com/tidymodels/recipes/issues/289)
+*  [`step_knnimpute()`](https://tidymodels.github.io/recipes/reference/step_knnimpute.html) can now pass two options to the underlying knn code, including the number of threads ([#323](https://github.com/tidymodels/recipes/issues/323)). 
+* Due to changes by CRAN,  [`step_nnmf()`](https://tidymodels.github.io/recipes/reference/step_nnmf.html) only works on versions of R >= 3.6.0 due to dependency issues. 
+*  [`step_dummy()`](https://tidymodels.github.io/recipes/reference/step_dummy.html) and [`step_other()`](https://tidymodels.github.io/recipes/reference/step_other.html) are now tolerant to cases where that step's selectors do not capture any columns. In this case, no modifications to the data are made. ([#290](https://github.com/tidymodels/recipes/issues/290), [#348](https://github.com/tidymodels/recipes/issues/348))
```

Comment:
```suggestion
*  [`step_dummy()`](https://tidymodels.github.io/recipes/reference/step_dummy.html) and [`step_other()`](https://tidymodels.github.io/recipes/reference/step_other.html) are now tolerant to cases where the step's selectors do not capture any columns. In this case, no modifications to the data are made. ([#290](https://github.com/tidymodels/recipes/issues/290), [#348](https://github.com/tidymodels/recipes/issues/348))
```

Reply:
^^ dealer's choice on the above, I think the is slightly more natural-sounding, but ¯\\(°_o)/¯